internal/cmd/cmd_sync_test.go

```diff
@@ -94,14 +94,14 @@ func (c *FakeClient) NewPublish(ctx context.Context) (gw.Publish, error) {
 	return &c.publishes[len(c.publishes)-1], nil
 }
 
-func (c *FakeClient) GetPublish(id string) gw.Publish {
+func (c *FakeClient) GetPublish(ctx context.Context, id string) (gw.Publish, error) {
 	for idx := range c.publishes {
 		if c.publishes[idx].id == id {
-			return &c.publishes[idx]
+			return &c.publishes[idx], nil
 		}
 	}
-	// Didn't find any, then return a broken one
-	return &BrokenPublish{id: id}
+	// Didn't find any, then return nil
```

Comment:
This behavior seems incorrect (not a good fake).

With the *real* exodus-gw client, if you do GetPublish on an ID of a nonexistent publish, it returns an error, doesn't it?  (returns `(nil, <some non-nil error>)`)

In that case the fake should do the same.

Some of the code relies on this, e.g. in internal/cmd/exodus.go there is:

```
		publish, err = gwClient.GetPublish(ctx, args.Publish)
		if err != nil {
			logger.F("error", err).Error("can't join publish")
			return 67
		}
		logger.F("publish", publish.ID()).Info("Joining publish")
```

If tested while providing a nonexistent publish ID, err will be nil, then won't this simply crash/panic at the publish.ID() expression?  (If this doesn't show up in any test, it's possible we just don't ever test the "ID of nonexistent publish" case using the fake client.)